.gitignore

```diff
@@ -9,3 +9,13 @@ prime/
 stage/
 snap/.snapcraft/
 snap/snapcraft.yaml
+snap-build/amd64/log
```

Comment:
Could you simplify to something like the following to avoid hard-coding the list of architectures?

```
snap-build/*/log
snap-build/*/*.img
```